src/modules/navigator/rtl_params.c

```diff
@@ -185,3 +174,32 @@ PARAM_DEFINE_FLOAT(RTL_LOITER_RAD, 50.0f);
  * @group Return Mode
  */
 PARAM_DEFINE_INT32(RTL_HDG_MD, 0);
+
+/**
+ * RTL time estimate safety margin factor
+ *
+ * Safety factor that is used to scale the actual RTL time estiamte.
+ * Time with margin = RTL_TIME_FACTOR * time + RTL_TIME_MARGIN
+ *
+ * @min 1.0
+ * @max 2.0
+ * @decimal 1
+ * @increment 0.1
+ * @group Return To Land
+ */
+PARAM_DEFINE_FLOAT(RTL_TIME_FACTOR, 1.1f);
+
+/**
+ * RTL time estimate safety margin offset
+ *
+ * Margin that is added to the time estimate, after it has already been scaled
+ * Time with margin = RTL_TIME_FACTOR * time + RTL_TIME_MARGIN
+ *
+ * @unit s
+ * @min 0
+ * @max 300
```

Comment:
300s sounds quite low as max, for vehicle flying multiple hours I would for sure want a bigger margin.
Would propose 3600 as max.

Reply:
https://github.com/PX4/PX4-Autopilot/pull/18711